### ISSUE TITLE
Used postgres version

### ISSUE THREAD

**Issue by mainrs** [tshatrov/ichiran#27]
Hello!

I am trying to convert the database to SQLite to get a better understanding on how it works and to try to embed it. What Postgres version do you use exactly?

**Comment by tshatrov** [comment]
Yeah, the locale name can be completely different. I imported the pg dump into Windows based postgres where the locale is called something else. The important thing is to create the database with the locale that is supported by your system, and then pg_restore into it. The locale name stored in the dump shouldn't matter.

**Comment by mainrs** [comment]
Oh, I didn't know that! I'll try that out later on! Thanks for the help!
